mrchecker/README.asciidoc

```diff
@@ -0,0 +1,87 @@
+= MrChecker under ProductionLine
+
+image::./images/mrchecker&pl.JPG[500,400]
+
+== Introduction
+MrChecker is end to end automation test framework written in Java. It has been released
+by Devon team which also is responsible for support of this framework. +
+
+This framework consist of eight test modules:
+
+* Core test module +
+* Selenium test module +
+* WebAPI test module +
+* Security test module +
+* DataBase test module +
+* Standalone test module +
+* DevOps module +
+
+== Prerequisites
+To be able to run Jenkins MrChecker job under ProductionLine you need to configure below settings in Jenkins and Gitlab
+
+* Jenkins +
+** Install required plugins: +
+HTTP Request Plugin +
+Allure Jenkins Plugin +
+** In Jenkins Global Tool Configuration configure Allure Commandline and Maven like +
+
+image::./images/allure.JPG[500,400]
+image::./images/maven.JPG[500,400]
+
+
+* Gitlab +
+** Generate User Private Token +
+Go to your Profile in Gitlab +
+
+image::./images/profile.png[500,400]
+
+Next click on the pen icon +
+
+image::./images/pen.png[500,400]
+
+On the left menu choose Access Tokens and put token name and check fields like below +
+
+image::./images/token.JPG[600,500]
+
+Click "Create personal access token", you should receive notification about created token and token string. Copy the token string.
+
```

Comment:
I think the group permission is missing.

Reply:
Was working without group permissions.

Reply:
In an empty instance it will not work. You can take the documentation from the devon4j-mts doc.